app/assets/javascripts/components/student/student_comments.es6.jsx

```diff
@@ -78,6 +68,43 @@ class StudentComments extends Component {
     };
   }
 
+
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+  componentWillMount() {
+    this.setState(StudentCommentsStore.getState());
+  }
+
+  componentDidMount() {
+    this._listener = StudentCommentsStore.listen((state) => this.setState(state));
+    StudentCommentsActions.fetchStudentComments(this.props.id);
+  }
+
+  componentWillUnmount() {
+    StudentCommentsStore.unlisten(this._listener);
+  }
+
+  // --------------------------------------------------
+  // Helpers
+  // --------------------------------------------------
+  showCreateOverlay() {
+    StudentActions.storeOverlay(true, 'create_comment');
+  }
+
+  // --------------------------------------------------
+  // Render
+  // --------------------------------------------------
+  renderStudentComment(comment, index) {
+    return (
+      <StudentComment comment={comment} key={index} />
```

Comment:
Let's give these props some `enter`s before them since we've been doing that everywhere else!